app/models/chat_channel.rb

```diff
@@ -44,7 +44,7 @@ def invite_only?
   end
 
   def clear_channel
-    messages.delete_all
+    messages.destroy_all
```

Comment:
This is necessary because not chat_channels and messages are bound with a foreign key constraint, thus messages with no chat channel id are no longer valid (and existing ones have been removed prior to those foreign keys migrations anyway)